ArduPlane/mode_avoidADSB.cpp

```diff
@@ -11,3 +11,8 @@ void ModeAvoidADSB::update()
     plane.mode_guided.update();
 }
 
+void ModeAvoidADSB::_navigate()
+{
+    plane.mode_loiter.navigate();
```

Comment:
I'd like them to call plane.update_loiter() instead. The problem with calling into mode_loiter is it means we're calling a member function on a class wirthout ever calling the init function on the class. Right now this is safe, but if someone changes mode_loiter it could break other modes.
In general I don't like calls into non-static functions on a mode class without the class enter function having been called

Reply:
yes, I totally agree with Tridge's comment.

Reply:
Changed to call into plane.update_loiter().